packages/gatsby/src/utils/webpack.config.js

```diff
@@ -229,10 +229,21 @@ module.exports = async (
             plugins.eslintGraphqlSchemaReload(),
           ])
           .filter(Boolean)
+        if (process.env.GATSBY_EXPERIMENTAL_DEV_SSR) {
+          // Don't use the default mini-css-extract-plugin setup as that
+          // breaks hmr.
+          configPlugins.push(
+            plutins.extractText({ filename: `[name].css` }),
```

Comment:
```
 ERROR 

plutins is not defined



  ReferenceError: plutins is not defined
  
  - webpack.config.js:236 getPlugins
    [gatsby-starter-blog]/[gatsby]/src/utils/webpack.config.js:236:13
```

:D

```suggestion
            plugins.extractText({ filename: `[name].css` }),
```

Reply:
heh, that's what happens kids when you don't test your code changes